docs/guide/pug-specific-options.md

```diff
@@ -465,3 +465,46 @@ Disables id formatting.
 ```pug
 foo(id="bar")
 ```
+
+---
+
+## Explicit div
+
+`pugExplicitDiv`
+
+### Description
+
+Define if a `div` tag should _always_ be printed with literal class and id formatting.
+
+### Options
+
+**Type:** `boolean`  
+**Default:** `false`
+
+#### `false`
+
+Input:
+
+```pug
+div.foo.bar
+```
+
+Output:
+
+```pug
+.foo.bar
+```
+
+#### `true`
+
+Input:
+
+```pug
+div.foo.bar
```

Comment:
```suggestion
.foo.bar
```

To make it even more clear, we should remove the tag here as it will be added if it doesn't exist

Otherwise we would need something like always, as-is and never
But I think this will never be requested by someone, and IMO it doesn't make sense to provide an option like as-is

Reply:
Agreed 😄 - the whole point of Prettier is to make code consistent so, always on or always off makes sense - a mix seems like the wrong direction. 👍